### DISCUSSION TITLE
Issue with adding a new language to OpenMetadata

### DISCUSSION THREAD

**Discussion by Snive6** [open-metadata/OpenMetadata#17361] [Q&A] [answered]
Hi,

I'm experiencing an unusual issue while setting up OpenMetadata on Kubernetes. I deployed it using the Helm chart as per the documentation. After deployment, I couldn’t locate the language JSON files, which I believe should be under /opt/openmetadata. I used both openmetadata/server and docker.getcollate.io/openmetadata/server images (version 1.4.5). Despite mounting a PV and PVC with my language file to /opt/openmetadata and modifying i18nextUtil.ts, the file doesn’t load. Could you please assist me in resolving this issue?

**Comment by harshach** [comment, marked as answer]
@Snive6 language files won't be updated in the release artifacts we ship. You need to provide contribution to the file in github repo.
Follow the docs here https://docs.open-metadata.org/v1.4.x/developers/how-to-add-language-support
